src/App.js

```diff
@@ -3,45 +3,58 @@ import { Switch, Route } from 'react-router-dom';
 import  { Homepage }  from '../src/Homepage/Homepage.js';
 import  { WordSelector }  from '../src/WordSelector/WordSelector.js';
 import  { Gamepage }  from '../src/Gamepage/Gamepage.js';
+import theCalls from '../src/APICalls/APICalls'
 import './App.css';
 
 export class App extends Component{
   constructor() {
     super()
-    this.state = {}
+    this.state = {
+      isGuesser: null,
+      word: '',
+      guess: ''
+    }
   }
-
-  componentDidMount() {
-
+  designateRole = async (role) => {
+    // const isGuesser = await theCalls.joinGame(role)
+    // this.setState({isGuesser})
+    console.log('designateRole')
+  }
+  makeWordToGuess = async (createdWord) => {
+    // const word = await theCalls.createWord(createdWord, 1)
+    // this.setState({word})
+    console.log("makeWordToGuess")
+  }
+  makeGuess = async (newGuess) => {
+    // const guess = await theCalls.makeGuess(newGuess)
+    // this.setState({guess})
+    console.log('makeGuess')
   }
```

Comment:
hmm, I don't know if you needed to merge this stuff. I guess we should be better at communicating, but this is where Chris and I left off and I had asked him to push up his branch without merging it so we could take a look at it later/put it in it's own PR.

Reply:
JK, I'm being a dumb. We definitely did talk about this. Disregard previous comment.